koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -622,11 +628,28 @@ def createContainer(self, src=None, target_info=None, arches=None,
         except (AttributeError, OsbsOrchestratorNotEnabled):
             # Older osbs-client, or else orchestration not enabled
             create_build_args['architecture'] = arch = arches[0]
+            create_build_args.pop('skip_build', None)
             create_method = self.osbs().create_build
             self.logger.debug("Starting %s with params: '%s'",
                               create_method, create_build_args)
             build_response = create_method(**create_build_args)
 
+        if build_response is None:
+            self.logger.debug("Build was skipped")
+
+            osbs_logs_dir = self.resultdir()
+            koji.ensuredir(osbs_logs_dir)
+            pid = os.fork()
+            if pid:
+                try:
+                    self._incremental_upload_logs(pid)
```

Comment:
Do we need the fork here? Since we are not waiting for build logs to be written, could we use koji.daemon.incremental_upload directly, or refactor `_incremental_upload_logs` to have a child pid as an optional param (or to behave differently when pid==0)?

Reply:
refactored and got rid of fork